src/set.js

```diff
@@ -0,0 +1,72 @@
+export default class Set {
+  constructor( arrayOfElements ) {
+    let _items = arrayOfElements || []
+    let _size = arrayOfElements ? arrayOfElements.length : 0
+
+    this.getItems = function() {
+      return _items
+    }
+    this.setItems = function( newItems ) {
+      _items = newItems
+      _size = _items.length
+    }
+
+    this.getSize = function() {
+      return _size
+    }
+    this.incrementSize = function() {
+      _size++
+    }
+    this.decrementSize = function() {
+      _size--
+    }
+  }
+
+  add( element ) {
+    let items = this.getItems()
+    if ( this.contains( element ) ) {
+      return false
+    } else {
+      items.push( element )
+      this.setItems( items )
+    }
+  }
+
+  remove( element ) {
+    let items = this.getItems()
```

Comment:
you should call `this.getItems()` when you know you need it. You could be calling it here only to return out on the next line. Maybe move `this.getItems()` down 2 lines?